StayFit/Signup/SignupViewController.m

```diff
@@ -169,6 +134,19 @@ - (void)viewController:(GMSAutocompleteViewController *)viewController
 didAutocompleteWithPlace:(GMSPlace *)place {
   [self dismissViewControllerAnimated:YES completion:nil];
     self.addressField.text = place.name;
+    for ( GMSAddressComponent * component in place.addressComponents) {
+        //checking the array to see where locality is located
+        if ( [component.type  isEqual: @"locality"] )
+        {
+            //automatically sets the name of the city
+            self.city.text = component.name;
+        }
+        if ( [component.type  isEqual: @"administrative_area_level_1"] )
```

Comment:
what is administrative_area_level_1? is this some field from google maps api

Reply:
yes, it is one of the types of field in the array that they gave us

Reply:
> what is administrative_area_level_1? is this some field from google maps api

yes, it is one of the types of field in the array that they gave us